packages/scripts/modules/known-paths.js

```diff
@@ -20,4 +21,9 @@ module.exports = {
    * Top level packages directory
    */
   packages,
+
+  /**
+   * Temporary files directory
+   */
+  tmp,
```

Comment:
Can we rename this to 'does not exist' or something? It's weird we have this in knownPaths even though we don't use it in implementation code?